### PR TITLE
Add GCP Marketplace to technical-specifications and release

### PR DESCRIPTION
**Pull request by lucacome**
### Proposed changes
Adds GCP listings to `docs/content/technical-specifications.md` and to release notes

### CODE REVIEW

File: docs/content/technical-specifications.md [hunk 1]
```diff
@@ -85,6 +85,15 @@ We also provide NGINX Plus images through the AWS Marketplace. Please see [Using
 |Debian-based image with NGINX App Protect WAF and DoS | ``debian:11-slim`` | NGINX App Protect WAF and DoS, NGINX Plus JavaScript and OpenTracing modules, OpenTracing tracers for Jaeger, Zipkin and Datadog | [F5 NGINX Ingress Controller with F5 NGINX App Protect DoS](https://aws.amazon.com/marketplace/pp/prodview-sghjw2csktega) | amd64 |
 {{% /table %}}
 
+We also provide NGINX Plus images through the Google Cloud Marketplace. Please see [Using GCP Marketplace Ingress Controller](/nginx-ingress-controller/installation/using-gcp-marketplace-package/) for details on how use them.
```
Comment (shaun-nx) [inline docs/content/technical-specifications.md:88]:
```suggestion
We also provide NGINX Plus images through the Google Cloud Marketplace. Please see [Using GCP Marketplace Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/using-gcp-marketplace-package/) for details on how use them.
```

**Review by shaun-nx — CHANGES_REQUESTED** [review]

**Review by ADubhlaoich — APPROVED** [review]
LGTM!

File: docs/content/technical-specifications.md (hunk 1 above)
Comment (vepatel) [reply docs/content/technical-specifications.md:88]: @shaun-nx I don't think a fully qualified ref is needed in this case as the ref is internal, eg: https://github.com/nginxinc/kubernetes-ingress/blob/v3.1.1/docs/content/installation/installation-with-manifests.md?plain=1#L19

**Review by vepatel — APPROVED** [review]

File: docs/content/technical-specifications.md (hunk 1 above)
Comment (shaun-nx) [reply docs/content/technical-specifications.md:88]: @vepatel Ah I see, thanks for noticing that!

**Review by shaun-nx — APPROVED** [review]

**Pull request merged**